### DISCUSSION TITLE
TitleBar comes up delayed

### DISCUSSION THREAD

**Discussion by hxhgxy** [ControlzEx/ControlzEx#184] [Q&A]
When I set NCActiveBrush to a color, for example, green, when window is loading, we see a blank (white) window for about 0.5s, and then the titlebar is shown up.
Is it possible to make the titlebar comes out immediately together with the window?

Thanks!

**Comment by batzen** [comment]
Are you on some Windows 11 version prior to 22H2 with a patch level from before July?
The blank/white window issue is caused by windows itself.

**Comment by hxhgxy** [comment]
Win 11 22H2 22621.2134.

**Reply by batzen** [reply]
Hmm. That's strange.
Does this also happen with the showcase application of ControlzEx?
